modules/rubiconBidAdapter.js

```diff
@@ -532,7 +532,7 @@ function parseSizes(bid) {
   if (hasVideoMediaType(bid)) {
     let size = [];
     if (typeof utils.deepAccess(bid, 'mediaTypes.video.playerSize') !== 'undefined') {
```

Comment:
If we are expecting an array this should be changed to check result with `Array.isArray`

Reply:
You are right, will add that checking